### ISSUE TITLE
Upload types?

### ISSUE THREAD

**Issue by vrdriver** [avikj/fbash#6]
Hi, 
This is not an issue, but more of a question of functionality.
Does this combination of the facebook-chat-api and fbash allow for the uploading of other attachments?
I'm considering this for uploading small mp3 files.
Just curious if you know before I go through the process of installing it.
Cheers,
Steve

**Comment by avikj** [comment]
Hi Steve,
You can currently use the `savefile` command with any non-image filetype. Here's an example with a .mp3 file:

![asdf](https://i.gyazo.com/63521589d362e1ca4725ebd489ef0231.png)

After uploading your file to Messenger, just run `savefile <filename>` to save it in fbash's current directory with that filename. Let me know if this answers your question.

**Comment by vrdriver** [comment]
Fantastic. I was wanting to use a system like this (I may have to modify it slightly) to be able to automatically upload files and use the FB platform as the delivery platform. It'll just be a one way system at this stage. Thanks.

**Comment by vrdriver** [comment]
Hi Avikj,
I managed to get it working and I can get a file to come through to me, but I can't save it on the computer I'm issuing the commands from.
So while it's an audio file, it's coming through as an audio file with a play button. 
However, I've also discovered that if I just zip the file and then send it, it comes through as a downloadable attachment at the browser end.
While I know you know how all these things work, it may be worth adding the requirements to getting the system working. eg, the facebook-chat-api, the latest version of nodejs and even possibly git? Just a thought.
Cheers,
Steve
![screen shot 2016-06-30 at 11 42 45](https://cloud.githubusercontent.com/assets/5615487/16474184/d5f4791a-3eb7-11e6-9534-f85317621787.png)
